app-ios/tutanota/Sources/Contacts/StructuredContactTypes.swift

```diff
@@ -0,0 +1,48 @@
+import Contacts
+
+enum ContactAddressType: String {
```

Comment:
[See the protocol comment](https://github.com/tutao/tutanota/pull/6437/files#r1469418529).